app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -0,0 +1,326 @@
+// Most of this is taken from the service manual and at some point needs to be
+// separated out into a component.  We are currently holding off on this until
+// it is tested more within the current navigation changes.
+
+window.GOVUK.Modules = window.GOVUK.Modules || {};
```

Comment:
Is there any particular reason we're now explicitly including this JavaScript file? Have you made any changes?